ironfish-cli/src/commands/chain/export.ts

```diff
@@ -49,14 +49,23 @@ export default class Export extends IronfishCommand {
 
     const client = await this.sdk.connectRpc()
 
+    const argStart = args.start as number | null
+    const argStop = args.stop as number | null
     const stream = client.exportChainStream({
-      start: args.start as number | null,
-      stop: args.stop as number | null,
+      start: argStart,
+      stop: argStop,
     })
-
     const { start, stop } = await AsyncUtils.first(stream.contentStream())
+    if (
+      (argStart && argStart < Number(GENESIS_BLOCK_SEQUENCE)) ||
+      (argStop != null && argStop > stop)
+    ) {
+      this.error(
+        `You can can only export chain from ${Number(GENESIS_BLOCK_SEQUENCE)} -> ${stop}`,
+      )
+      this.exit(1)
```

Comment:
This will break valid uses of the export command: negative values for start or end export blocks relative to the end of the chain.

Reply:
Got it, but I guess we may still want some check on the upper end ? For example chain is at 100 and someone requested upto 200, in that case do we want a message to tell the users what the highest they can request ?

If yes I can fix this PR to have only that change

Reply:
Okay Sure

Reply:
Updated the PR to reflect the same